### PR TITLE
Spelling and Grammar fixes

### PR DESCRIPTION
**Pull request by SeanTAllen**

### CODE REVIEW

File: testing/notifier-interactions.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Testing
+
+Most of us are familiar with unit testing code with more traditional concurrency models. In this case of threaded code, for many, this means throwing up your hands in disgust and walking away. Testing concurrency with threads is hard. Testing concurrency is Pony is much easier but if you haven't encountered anything like it before it can be hard to know where to start.
```
Comment (Theodus) [inline testing/notifier-interactions.md]: "Testing concurrency is Pony is much easier" -> "Testing concurrency in Pony is much easier"

Comment (Theodus) [inline testing/notifier-interactions.md]: Add a comma after "much easier but"

Comment (SeanTAllen) [inline testing/notifier-interactions.md]: done

File: testing/notifier-interactions.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# Testing
+
+Most of us are familiar with unit testing code with more traditional concurrency models. In this case of threaded code, for many, this means throwing up your hands in disgust and walking away. Testing concurrency with threads is hard. Testing concurrency in Pony is much easier but if you haven't encountered anything like it before it can be hard to know where to start.
```
Comment (Theodus) [inline testing/notifier-interactions.md]: There should be a comma after the "but" in "Testing concurrency in Pony is much easier but".

Comment (SeanTAllen) [inline testing/notifier-interactions.md]: grammar checker says either is acceptable but i updated anyway

**Pull request merged**